data/persistence/src/androidMain/AndroidManifest.xml

```diff
@@ -1,3 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="social.androiddev.data.persistence" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" />
```

Comment:
Was already defined in the gradle file.

Reply:
Good catch - we shouldn't be setting this in manifest anymore regardless